pages/dashboard/tracing/[caseId]/[contactId]/monitor.tsx

```diff
@@ -29,8 +36,68 @@ export default function Monitor() {
             <h1 className="text-2xl font-semibold">Tambah Pemantauan</h1>
           </div>
         </div>
-      </ConfirmedCasesHeader>
-      <DashboardPageContent>Content</DashboardPageContent>
+      </DashboardPageHeader>
+      <DashboardPageContent>
+        <div className="space-y-4">
+          <SectionCard className="space-y-6">
+            <div className="space-y-4">
+              <SectionCardHeader title="INFORMASI UTAMA" />
```

Comment:
```suggestion
              <SectionCardHeader title="Informasi Utama" />
```

Don't use all caps in HTML content. Always modify the capitalisation state through CSS.